include/swift/Basic/Mangler.h

```diff
@@ -123,6 +123,11 @@ class Mangler {
   /// Appends a mangled identifier string.
   void appendIdentifier(StringRef ident);
 
+  // NOTE: the addSubsitution functions perform the value computation before
+  // the assignment because there is no sequence point synchronising the
+  // computation of the value before the insertion of the new key, resultin in
```

Comment:
Typo:
```suggestion
  // computation of the value before the insertion of the new key, resulting in
```

Reply:
*sigh* one of these days, I will learn to type.  Thank you @theblixguy for catching this.